### PR TITLE
use the correct directionality for the graph and then use the graph only for pinning

### PR DESCRIPTION
**Pull request by CJ-Wright**
This needs a fix to the graph (namely the PRs "issued" against the pinning)

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@regro/auto-tick

**Comment by beckermr** [comment]
LGTM. I guess we have to fix the graph too.

**Comment by CJ-Wright** [comment]
For some reason this is still putting out a node for the pinning.

**Comment by CJ-Wright** [comment]
I'll debug tomorrow

**Comment by CJ-Wright** [comment]
@beckermr

File: conda_forge_tick/migrators/core.py [hunk 1]
```diff
@@ -568,10 +568,14 @@ def _test_node_built(node):
 
             return True
 
+        # Special case pinning so we don't need to add it as a dep for every feedstock
```
Comment (beckermr) [inline conda_forge_tick/migrators/core.py:571]: This comment is wrong. I think you mean to say we don't need to add every feedstock to its deps.

**Comment by beckermr** [comment]
One comment here, but otherwise LGTM.

File: conda_forge_tick/migrators/core.py (hunk 1 above)
Comment (CJ-Wright) [inline conda_forge_tick/migrators/core.py:571]:
```suggestion
        # Special case pinning so we don't need to add every feedstock as a dep
```

**Comment by CJ-Wright** [comment]
Ready for another round

**Comment by beckermr** [comment]
what is your plan for fixing the graph? bot re-run labels? adjust the json by hand?

**Comment by CJ-Wright** [comment]
For the PRs that we actually issued re-runs should be ok. For those we should have issued but didn't (eg. root) I will go in and delete the offending metadata.

**Pull request merged**